### PR TITLE
Remove 5.x obsolete options checks

### PR DESCRIPTION
**Pull request by ph**
Since we are moving to 7.0 its a good time to update the code and remove
deprecated code and warning for removed options.

Remove any 5.x options
Add tests for obsolete Filebeat 6.x options

### CODE REVIEW

**Comment by ph** [comment]
The docker error seems to be a problem with our infra, it should be resolved soon.

File: libbeat/common/cfgwarn/removed_test.go [hunk 1]
```diff
@@ -123,14 +107,26 @@ func TestRemovedSettings(t *testing.T) {
...
+		},
 	}
 
 	functions := []struct {
 		name string
 		fn   func(*common.Config, ...string) error
 	}{
 		{name: "checkRemovedSetting", fn: checkRemovedSettings},
-		{name: "checkRemoved6xSetting", fn: CheckRemoved6xSettings},
+		// {name: "checkRemoved6xSetting", fn: CheckRemoved6xSettings},
```
Comment (kvch) [inline libbeat/common/cfgwarn/removed_test.go]: Is this comment a leftover?

**Comment by ph** [comment]
@kvch updated with the uncomment code, I have fat fingers :(

**Comment by ph** [comment]
Jenkins test this please

**Pull request merged**